.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Add vscode related things to .gitignore?

Reply:
@wanghaoran1988 I debated it, but the config directive here does make sense. But, now that you mention it, I think we'll land into trouble if people have conflicting settings. I'll go ahead and exclude it.